tests/settings/utils_test.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (C) 2017  Lucas Hoffmann
+# This file is released under the GNU GPL, version 3 or a later revision.
+# For further details see the COPYING file
+
+from __future__ import absolute_import
+
+import unittest
+
+import mock
+
+from alot.settings import utils
+
+
+class TestResolveAtt(unittest.TestCase):
+
+    __patchers = []
+    fallback = mock.Mock()
+    fallback.foreground = 'some fallback foreground value'
+    fallback.background = 'some fallback background value'
+
+    @classmethod
+    def setUpClass(cls):
+        cls.__patchers.append(mock.patch('urwid.AttrSpec', mock.Mock(
```

Comment:
This is your problem. The matra you need when using mock is "mock it where you use it", so in this case what you actually want to patch is `'alot.settings.utils.AttrSpec'`.

the long version of this is that python makes a shallow copy to prevent people doing stupid things from affecting other people by doing a shallow copy of imports into your module.